src/open_inwoner/accounts/views/profile.py

```diff
@@ -224,6 +226,31 @@ def get_form_kwargs(self):
         return kwargs
 
 
+class MyCategoriesView(
+    LogMixin, LoginRequiredMixin, CommonPageMixin, BaseBreadcrumbMixin, UpdateView
+):
+    template_name = "pages/profile/categories.html"
+    model = User
+    form_class = CategoriesForm
+    success_url = reverse_lazy("profile:detail")
+
+    @cached_property
+    def crumbs(self):
+        return [
+            (_("Mijn profiel"), reverse("profile:detail")),
+            (_("Mijn interessegebieden"), reverse("profile:categories")),
```

Comment:
I think so, because the `onderwerpen` on the homepage are filled with highlighted categories by default (and not always selected manually by the user as interests)